administrator/components/com_joomgallery/controllers/config.php

```diff
@@ -63,6 +63,14 @@ public function __construct()
    */
   public function edit()
   {
+    $id  = JRequest::getInt('id');
+    $cid = JRequest::getVar('cid', array(), 'post', 'array');
+
+    if(!$id && count($cid) && $cid[0])
+    {
+      JRequest::setVar('id', (int) $cid[0]);
+    }
+
     JRequest::setVar('view', 'config');
     JRequest::setVar('hidemainmenu', 1);
 
```

Comment:
I have inserted following code after line 63 in config.php:

` 
   */


     public function edit()
   {
+    $id  = JRequest::getInt('id');
+    $cid = JRequest::getVar('cid', array(), 'post', 'array');
+
+    if(!$id && count($cid) && $cid[0])
+    {
+      JRequest::setVar('id', (int) $cid[0]);
+    }
+
     JRequest::setVar('view', 'config');
     JRequest::setVar('hidemainmenu', 1);



  /**

`

When opening configuration of joomgallery I have got a white monitor.